package.json

```diff
@@ -79,12 +80,13 @@
         "rc-select": "^14.1.14",
         "react-phone-number-input": "3.1.41",
         "rollup-plugin-scss": "^4.0.0",
-        "supertokens-js-override": "^0.0.4",
-        "supertokens-web-js": "^0.3.0"
+        "supertokens-js-override": "^0.0.4"
     },
     "peerDependencies": {
         "react": ">=16.8.0",
-        "react-dom": ">=16.8.0"
+        "react-dom": ">=16.8.0",
+        "supertokens-web-js": "^0.3.0",
+        "supertokens-website": "supertokens/supertokens-website#feat/getWindowUnsafe"
```

Comment:
why do you need to add supertokens-website to the peer dependency? Shouldn't web-js already have that as a dependency?

Reply:
web-js has the dependency, but we also import it directly here. while I think this should be fixed, right now, it'd result in the website SDK being bundled. If you prefer, I can remove this and add the website SDK into the bundler config as an external dependency, but it shouldn't be bundled.